Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -465,17 +465,36 @@ public async Task RunStartupTasksAsync()
 
             var stopWatch = new Stopwatch();
             stopWatch.Start();
-            await Task.WhenAll(StartEntryPoints(entryPoints, true)).ConfigureAwait(false);
+            await Task.WhenAll(StartEntryPointsSafe(entryPoints, true)).ConfigureAwait(false);
             Logger.LogInformation("Executed all pre-startup entry points in {Elapsed:g}", stopWatch.Elapsed);
 
             Logger.LogInformation("Core startup complete");
             CoreStartupHasCompleted = true;
             stopWatch.Restart();
-            await Task.WhenAll(StartEntryPoints(entryPoints, false)).ConfigureAwait(false);
+            await Task.WhenAll(StartEntryPointsSafe(entryPoints, false)).ConfigureAwait(false);
             Logger.LogInformation("Executed all post-startup entry points in {Elapsed:g}", stopWatch.Elapsed);
             stopWatch.Stop();
         }
 
+        private IEnumerable<Task> StartEntryPointsSafe(IEnumerable<IServerEntryPoint> entryPoints, bool isBeforeStartup)
+        {
+            IEnumerable<Task> value;
+            try
+            {
+                value = StartEntryPoints(entryPoints, isBeforeStartup);
+            }
+            catch (Exception ex)
+            {
+                Logger.LogError(ex, "Exception in implementing SafeEntyPoint interface.");
+                value = new List<Task>
```

Comment:
```suggestion
                value = new[]
```

No reason to create a list for this